src/locales/en/translation.json

```diff
@@ -194,8 +194,8 @@
   "epidemic.imported": "Imported",
   "epidemic.imported_close_contact": "Close contact of imported case",
   "epidemic.local": "Local",
-  "epidemic.local_close_contact": "Close contact of local case",
+  "epidemic.local_close_contact": "Close contact with local case",
   "epidemic.local_possibly": "Possibly local",
-  "epidemic.local_possibly_close_contact": "Close contact of possibly local case",
+  "epidemic.local_possibly_close_contact": "Close contact with suspected local case",
```

Comment:
For case classification, we follow the wordings from the HKSAR.
https://www.chp.gov.hk/files/pdf/local_situation_covid19_en.pdf